<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="Button_CleanTimer">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Clean Timer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_Kill">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Kill</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="Button_SetApp">
         <property name="text">
          <string>Run Appliction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_StartTimer">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Start Timer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>EveryRun Time(s)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="SpinBox_RunTime">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>604800</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>EveryBoot Waiting Time(s)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="SpinBox_Waiting">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Action Interval(s)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="SpinBox_ClickInterval">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Action Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="SpinBox_ActionNumber">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QCheckBox" name="CheckBox_SetPos">
         <property name="text">
          <string>F2 to Set Action Pos:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ComboBox_ClickNo">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="TextEdit_Info">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
